package.json

```diff
@@ -8,7 +8,8 @@
     "build:docs": "node scripts/generate-service-docs.js",
     "health:node-abi": "node -p \"process.versions.node + ' ABI ' + process.versions.modules\"",
     "prepare": "chmod +x ./bin/promethean.js",
-    "lint": "biome lint --config-path=biome.json .",
+    "lint": "eslint --cache ./packages",
+    "lint:diff": "eslint $(git --no-pager diff --name-only --diff-filter=d origin/main...HEAD -- '*.ts' '*.tsx') || echo 'No TS/TSX changes to lint.'",
```

Comment:
Added elint diff and switched (for now) from biome to eslint for linting.
It's the tool I know, and I need feedback now, before adding new toys.